nuxeo-features/nuxeo-automation/nuxeo-automation-core/src/main/java/org/nuxeo/ecm/automation/core/operations/blob/BlobToFile.java

```diff
@@ -57,6 +59,11 @@ protected void init() {
         root.mkdirs();
     }
 
+    protected boolean isTargetDirectoryForbidden() {
+        File nuxeoHome = ConfigurationGenerator.retrieveNuxeoHome();
+        return Paths.get(directory).startsWith(nuxeoHome.toPath());
```

Comment:
Use `Paths.get(directory).toAbsolutePath().normalize().startsWith(nuxeoHome.toPath().toAbsolutePath().normalize())` to avoid dangerous `..` path components that would allow escaping the nuxeoHome directory.